shlr/wind/iob_pipe.c

```diff
@@ -2,7 +2,9 @@
 
 #include <stdio.h>
 #include <string.h>
+#ifndef _MSC_VER
```

Comment:
probably iob_pipe doesnt needs unistd.h :? and this is a platform-specific include file, not compiler related